src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/AbstractDedupJobScheduler.java

```diff
@@ -86,27 +86,40 @@ public TimeJobInfo reserveSingle(final String tube, final long ttrInMillis) {
     }
 
     public long getReadyJobCount(String tube) {
-        return getJobCount(getReadyQueue(tube));
+        final String queue = getReadyQueue(tube);
+        final long now = Instant.now().getMillis();
+        return rdbi.withHandle(new Callback<Long>() {
+            @Override
+            public Long run(Handle handle) {
+                return handle.jedis().zcount(queue, 0, now);
```

Comment:
I grepped through other projects that use these schedulers - and looks like omega is the only one calling this method - which has new behavior.  I agree it is safe to change this implementation.